modules/main-rules/pages/transport.adoc

```diff
@@ -1,39 +1,61 @@
+
+You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
 Obviously, once you disembark them then the normal rules apply again.
 
+[NOTE]
+.Placement of disembarked units
+====
+When you disembark units, measure their movement from the base of the {transport} unit.
+If you do not have enough movement to clear the {transport} unit's base then place the disembarked unit in base contact with the {transport} unit.
+====
+
+[NOTE.e40k]
+.Placement of disembarked units
+====
+We added this rule to minimise the effects that result from the assortment of different base sizes that players use.
+====
+
 == Destruction of {transport} units
 
-If a {transport} unit is destroyed then, just before you remove the model as a casualty, roll a D6 for each embarked unit:
+If a {transport} unit is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit:
 
-* *1–3*: The embarked unit is destroyed along with the {transport} unit.
+* *1–3*: The embarked unit suffers a hit.
 * *4–6*: The embarked unit survives -- place it in contact with the destroyed {transport} unit.
 
-Then remove the {transport} unit model as normal.
+Then remove the {transport} unit as normal.
 
-== Hitching a lift (Orks)
+== Transportation by units that have the {hitch} ability
 
-Ork infantry units can 'hitch a lift' on some Ork {transport} units -- they simply grab onto the outside of the vehicle and hang on for a crazy ride over the battlefield.
+Units that have the {hitch} ability can carry Infantry units across the battlefield -- they 'hitch a lift'.
+That is, they simply grab onto the outside of the vehicle and hang on for a crazy ride over the battlefield.
 
-* You may use this rule only where you embark Ork infantry units with *Battlewagon* and *Battle Fortress* units.
-Battlewagons can embark one infantry unit, Battle Fortresses can embark up to four.
-+[TODO: Consider a different approach to the text -- some later/future rules might allow other units or vehicles to participate in hitching a lift.]+
-* You can simply embark and disembark units as you move the vehicle -- there is no cost to the {transport} unit's movement.
-But you may not make any independent movement with the Ork infantry units in the same phase in which they hitch a lift.
-* A unit of Orks can only hitch a lift once per phase.
+To use the {hitch} ability to transport an Infantry unit:
 
-+[TODO: Consider whether to move this 'Orks only' rule to the special rules statements in the Ork army list. 
-Unless any later content expanded it to other races?
-Also, might need to add detail to clarify 'multiple move' situations (March orders).]+
+* You must embark *and* disembark your Infantry unit in the same phase.
+Embark the unit at any point during your move of the {hitch} unit, and disembark either at the end that same move -- or at the end of another move in the same phase, if the {hitch} unit is on March orders.
```

Comment:
This is a rules change, there are no restrictions on when the unit can disembark.

Maybe:

```suggestion
Embark the unit at any point during your move of the {hitch} unit, and disembark at another point during same move -- or during another move in the same phase, if the {hitch} unit is on March orders.
```

Reply:
Funny thing - I **swear** I saw something about 'at the end of the move' when I was revising the text, and that's why I worded it that particular way. But looking at the E40k source I don't see where it could've crept in from, if indeed it was there (I'll look at the history tomorrow just to satisfy my bafflement). I'm off to bed shortly, but good catch, I'll check your suggestion and fix tomorrow.

Reply:
Aha! It seems that I'm not entirely mad! :-D Previously, as from commit [e9e402b](https://github.com/GameBrains/er-core/pull/8/commits/e9e402becb9048a6b0c17e513b5f8ba133d158bb), it said:

`* Any embarked unit *must* disembark at the end of the {hitch} unit's move, or at the end of its last move during a March order.`

And... that was actually your commit, you bugger! ;-D

Prior to that it said:

`* You can simply embark and disembark units as you move the vehicle -- there is no cost to the {transport} unit's movement.`

Ah well, stuff happens. I'll look at your latest suggestion next.

Reply:
> And... that was actually your commit, you bugger! ;-D

Doh! Oops. :-(